src/models/u256_decimal.rs

```diff
@@ -0,0 +1,92 @@
+use primitive_types::U256;
+use serde::{de, Deserializer, Serializer};
+use serde_with::{DeserializeAs, SerializeAs};
+use std::fmt;
+
+// Code copied from here: https://github.com/cowprotocol/services/blob/main/crates/model/src/u256_decimal.rs
+// It was copied, as we prefer to not depend on such a big project.
+
+pub struct DecimalU256;
+
+impl<'de> DeserializeAs<'de, U256> for DecimalU256 {
+    fn deserialize_as<D>(deserializer: D) -> Result<U256, D::Error>
+    where
+        D: Deserializer<'de>,
+    {
+        deserialize(deserializer)
+    }
+}
+
+impl<'de> SerializeAs<U256> for DecimalU256 {
+    fn serialize_as<S>(source: &U256, serializer: S) -> Result<S::Ok, S::Error>
+    where
+        S: Serializer,
+    {
+        serialize(source, serializer)
+    }
+}
```

Comment:
Note that this part isn't used since you don't use `#[serde_as(as = ...)]` but instead `#[serde(with = ...)]`.

Might be worth keeping it around if you ever need:
```
struct Foo {
    #[serde_as(as = "Option<DecimalU256>")]
    value: Option<U256>
}
```